woocommerce/woocommerce
# Migrate `@woocmmerce/onboarding` to TS

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33132

This PR migrates @woocmmerce/onboarding to TS and fixes the related type errors in ./client.


### How to test the changes in this Pull Request:

1. Run `pnpm nx build @woocmmerce/onboarding `
2. Observe that `packages/js/onboarding/build-types` is generated

### Other information:




### FOR PR REVIEWER ONLY:
